packages/react-components/src/organisms/WorkingGroupsTabbedCard.tsx

```diff
@@ -0,0 +1,67 @@
+import { css } from '@emotion/react';
+import React from 'react';
+
+import { WorkingGroupMembership } from '@asap-hub/model';
+import { TabbedCard, WorkingGroupsList } from '../molecules';
+import { rem, tabletScreen } from '../pixels';
+import { splitListBy } from '../utils';
+
+const containerStyles = css({
+  padding: `${rem(32)} 0`,
+  display: 'grid',
+  rowGap: rem(12),
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    columnGap: rem(15),
+  },
+});
+
+type WorkingGroupsTabbedCardProps = {
+  userName: string;
+  groups?: ReadonlyArray<WorkingGroupMembership>;
+  isUserAlumni: boolean;
+};
+
+const WorkingGroupsTabbedCard: React.FC<WorkingGroupsTabbedCardProps> = ({
+  userName,
+  groups = [],
```

Comment:
I think there's an issue here. If `groups` is initially `undefined` because it hasn't loaded yet it will come as `[]` here, so in the tabbed card appears as if the person is not associated to any working groups. But if we refresh the page we can see that actually the person is associated with one

https://user-images.githubusercontent.com/16595804/211801595-49e2a581-d56a-449b-8efe-e0206fadb2c9.mov

with `return null` I believe we don't need to refresh the page :)